components/d2l-colors/d2l-colors.js

```diff
@@ -0,0 +1,130 @@
+			--d2l-color-citrine: #ffba59;
+			--d2l-color-citrine-minus-1: #a87b3b;
+			--d2l-color-citrine-minus-2: #694b25;
+
+			/* topaz */
+			--d2l-color-topaz-plus-1: #fffbb2;
+			--d2l-color-topaz: #f5ec5a;
+			--d2l-color-topaz-minus-1: #a69f21;
+			--d2l-color-topaz-minus-2: #635f15;
+
+			/* peridot */
+			--d2l-color-peridot-plus-1: #cdf2b1;
+			--d2l-color-peridot: #8ccd5a;
+			--d2l-color-peridot-minus-1: #59823a;
+			--d2l-color-peridot-minus-2: #3d5926;
+
+			/* olivine */
+			--d2l-color-olivine-plus-1: #a5fabd;
+			--d2l-color-olivine: #46a661;
+			--d2l-color-olivine-minus-1: #327846;
+			--d2l-color-olivine-minus-2: #275c36;
+
+			/* malachite */
+			--d2l-color-malachite-plus-1: #a1f7f4;
+			--d2l-color-malachite: #00afaa;
+			--d2l-color-malachite-minus-1: #017d79;
+			--d2l-color-malachite-minus-2: #005e5b;
+
+			/* primary accent */
+			--d2l-color-primary-accent-action: var(--d2l-color-celestine);
+			--d2l-color-primary-accent-indicator: var(--d2l-color-carnelian);
+
+			/* feedback */
+			--d2l-color-feedback-error: var(--d2l-color-cinnabar);
+			--d2l-color-feedback-warning: var(--d2l-color-citrine);
+			--d2l-color-feedback-success: var(--d2l-color-olivine);
+			--d2l-color-feedback-action: var(--d2l-color-celestine-plus-1);
+
+			/* deprecated */
+			--d2l-color-white: #fff;
+			--d2l-color-woolonardo: var(--d2l-color-sylvite);
+			--d2l-color-titanius: var(--d2l-color-mica);
+			--d2l-color-pressicus: var(--d2l-color-corundum);
+			--d2l-color-violettine: var(--d2l-color-amethyst-minus-1);
+			--d2l-color-panthera: var(--d2l-color-tourmaline-plus-1);
+			--d2l-color-koolaudica: var(--d2l-color-peridot);
+			--d2l-color-deephonica: var(--d2l-color-malachite);
+			--d2l-color-lurite: var(--d2l-color-topaz);
+			--d2l-color-buttonic: linear-gradient(to bottom, var(--d2l-color-regolith) 0%, var(--d2l-color-gypsum) 100%);
+			--d2l-color-meglor: linear-gradient(to bottom, var(--d2l-color-corundum) 0%, var(--d2l-color-tungsten) 100%);
+			--d2l-color-trancition: linear-gradient(to bottom, #fff 0%, var(--d2l-color-regolith) 100%);
+			--d2l-color-trixon: linear-gradient(to bottom, var(--d2l-color-regolith) 0%, var(--d2l-color-sylvite) 100%);
+			--d2l-color-azurite: var(--d2l-color-zircon-plus-1);
+			--d2l-color-celestuba: var(--d2l-color-celestine-minus-1);
+			--d2l-color-gravah: var(--d2l-color-amethyst-minus-1);
+			--d2l-color-saphirella: var(--d2l-color-zircon-plus-1);
+			--d2l-color-chartronic: var(--d2l-color-topaz);
+			--d2l-color-celestine-light-1: var(--d2l-color-celestine-plus-2);
+			--d2l-color-celestine-light-2: var(--d2l-color-celestine-plus-1);
+			--d2l-color-olivine-light-1: var(--d2l-color-olivine-plus-1);
+			--d2l-color-olivine-light-2: var(--d2l-color-olivine);
+			--d2l-color-zircon-light-1: var(--d2l-color-zircon-plus-2);
+			--d2l-color-zircon-light-2: var(--d2l-color-zircon-plus-2);
+		}
+	`;
+	head.appendChild(style);
+	window.ShadyCSS.CustomStyleInterface.addCustomStyle(style);
```

Comment:
This is needed in order for the global CSS properties to be resolve correctly in components when using IE11.  I think this was previously done for us in `custom-style`.  An alternative to this is to just put all these variables in a standard `.css` file and then rely on variable fallback value in IE11 - this is kinda crumby though because then the components need to have the values, and it's something that is easily forgotten.